libs/logic-apps-shared/src/utils/src/lib/models/template.ts

```diff
@@ -43,6 +43,7 @@ export interface Parameter {
 
 export interface ParameterDefinition extends Parameter {
   value?: any;
+  associatedWorkflows?: string[];
```

Comment:
Why do we have this list here? The manifest itself wont have any mention of this no?
We will have to determine based on the parameter key

Reply:
We could do that, but if we go that path, then everytime the parametersTab renders, we'd have to
   for every parameter
      for every manifest
         check if this parameter name exists, then added it to associatedWorkflows to display

I thought keeping it here as part of loading parameter process will be more runtime efficient

Reply:
We already have a way to do that.. from teh list of workflows in stores, we need to find each parameter exists in which workflow.parameterKeys... we dont have to look into manifest at all... i kept it that way for this reason

Reply:
resolved it as discussed offline